source/blog/2018-07-19-ovn-dpdk.html.md

```diff
@@ -65,14 +65,17 @@ For more information about OVN localnet integration in oVirt, refer to oVirt's [
     # ansible-galaxy install oVirt.dpdk-setup
     ```
 2) Execute the role as described above.
-3) Unless the host was already configured, the host will restart for kernel changes to be applied.
-4) Make sure the following:
+3) Unless the host was already configured, the host will restart for kernel changes to be applied. Note that after reboot the following values are changed (If they are not, you may have incompatible hardware, or you should report a bug on ovirt@users.org):
     * IOMMU: ```/sys/devices/virtual/iommu/<DMAR device>/devices/<PCI address>``` should exist. DMAR devices are typically annotated by dmar0, dmar1, and so forth.
     * hugepages: ```grep Huge /proc/meminfo``` should reflect the desired state as defined in the Ansible playbook (i.e. `Hugepagesize` and `HugePages_Total`)
```

Comment:
please add a word about NICs changing names.

Reply:
It would be nice to have a UI for this. I'm currently working on a project to expose Ansible roles / playbooks to oVirt UI:

https://trello.com/b/8UmL1LBD/rhv-ovirt-extensions

Let me know if you'd like help.

Reply:
i added you to the board.

Here's the project doc: https://docs.google.com/document/d/1n4VlosdPnIZL81wx0sEIrkAfBehuV5DfOU7eH9isWT0/edit